miqa/settings.py

```diff
@@ -78,6 +78,11 @@ class TestingConfiguration(MiqaMixin, TestingBaseConfiguration):
     CELERY_TASK_ALWAYS_EAGER = True
 
 
+class PyppeteerTestingConfiguration(MiqaMixin, DevelopmentBaseConfiguration):
+    # We would like to test that the celery tasks work correctly when triggered from the API
+    CELERY_TASK_ALWAYS_EAGER = True
```

Comment:
Does the regular `TestingConfiguration` not work for this?

Reply:
Excerpt from https://girder-pytest-pyppeteer.readthedocs.io/en/latest/setup/#add-the-test-pyppeteer-tox-environment:

>DJANGO_CONFIGURATION = DevelopmentConfiguration - This is a shortcut that uses a default [composed configuration](https://github.com/girder/django-composed-configuration) to conveniently set everything required for a more-or-less functioning server environment suitable for testing against. The TestingConfiguration omits several settings that are needed when running a realistic server, so the DevelopmentConfiguration is used instead. You may have an existing configuration that is more suitable, or you may want to set up your own custom PyppeteerTestingConfiguration for explicitly configuring the test environment.